release-notes.md

```diff
@@ -1,5 +1,70 @@
+
+```
+{
+  "env": {
+    "node": true,
+    "es6": true
+  },
+  "parserOptions": {
+    "ecmaVersion": 6
+  }
+}
+```
+
+If your project is using [medic-android](https://github.com/medic/medic-android) you must be running >v0.1.203 (March 2017) to use `ecmaVersion: 6`. The `medic-android` project does not support ecmaVersions above 6.
+
+### Breaking Change - Modules
+As part of our effort to leverage JavaScript standards, JavaScript configuration code should now use `require('./file')` or `import lib from './file';` to share code between multiple JavaScript files. The global `extras` object is no longer present and you may see the error `'extras' is not defined` when running `compile-app-settings`. This global object was a reference to `nools-extras.js` or `contact-summary-extras.js`, so you can achieve the same behavior by adding the line `var extras = require('./nools-extras');` or `import extras from './nools-extras';` to the top of your JavaScript.  This gives configuration authors the flexibility to have multiple library files, and enables the sharing of code across projects.
+
+You may also find it helpful to leverage third-party modules like [MomentJs](https://momentjs.com/) or [lodash](https://lodash.com/). You may install them using [npm](https://docs.npmjs.com/about-npm/) packages. For example, run `npm install moment` and then add `const moment = require('moment');` in your script. Ensure that you maintain a reasonable file size and memory footprint for your scripts. We support [tree shaking](https://webpack.js.org/guides/tree-shaking/) so use it when possible.
+
+### Breaking Change - contact-summary.js as an ES Module
+As part of our effort to leverage JavaScript standards and increase the testability of code, `contact-summary.js` should be a standard ES Module instead of using a bare return. This allows you to write standard unit tests for the file instead of needing a harness or custom wrapper.
+
+`contact-summary.js` in medic-conf 1.x or 2.x:
+```
+return {
+  fields: [],
+  cards: [],
+  context: {},
+};
+```
+
+`contact-summary.js` in medic-conf 3.0:
+```
+module.exports = {
+  fields: [],
+  cards: [],
+  context: {},
+};
+```
+
+### Breaking Change - Declarative Configuration library is now internal
+The [Declarative Configuration System](https://github.com/medic/medic-docs/blob/master/configuration/developing-community-health-applications.md) explicitly provides documented functions which are meant to be re-used by configuration code (eg. [Utils](https://github.com/medic/medic-docs/blob/master/configuration/developing-community-health-applications.md#utils)). It was also possible for configuration code to use the undocumented internals of the declarative library (commonly used examples are `now`, `isReportValid()`, and `emitTargetInstance()`). In medic-conf 3.0, all internal functions and variables are no longer accessible. Documented interfaces remain unchanged.
```

Comment:
This may not be an issue or incorrect, but I wanted to flag that the first sentence here is in present tense, the second in past tense, and the third in present again. As a lay person, this signals to me that:

> The [Declarative Configuration System](https://github.com/medic/medic-docs/blob/master/configuration/developing-community-health-applications.md) explicitly provides documented functions which are meant to be re-used by configuration code (eg. [Utils](https://github.com/medic/medic-docs/blob/master/configuration/developing-community-health-applications.md#utils)). 

is still correct / the case in medic-conf 3.0. If that is not true, I would recommend changing tenses.

Reply:
My intention is to have three sentences. 1) This is the intended ideal, 2) in contrast, this is how it was, 3) this is how it is now